resources/galera_configuration.rb

```diff
@@ -58,9 +58,36 @@
   end
 end
 
+action :bootstrap do
+  ruby_block 'bootstrap galera cluster' do
+    block do
+      shell_out!('galera_new_cluster')
```

Comment:
is this correct as it is a string?

Reply:
Yes, this is the command to be run which is in the path.